src/components/cc-env-var-create/cc-env-var-create.js

```diff
@@ -41,8 +42,8 @@ export class CcEnvVarCreate extends LitElement {
     /** @type {boolean} Sets `disabled` attribute on inputs and button. */
     this.disabled = false;
 
-    /** @type {string} Sets the mode of the variables name validation. */
-    this.mode = '';
+    /** @type {EnvVarMode} Sets the mode of the variables name validation. */
+    this.mode = 'simple';
```

Comment:
praise: nice touch to be stricter with a type instead of a string

suggestion: could we rename this generic mode (and it type) to something that refers to validation to disambiguate with the SIMPLE/EXPERT/JSON mode?

Proposition:

* `EnvVarMode` => `EnvVarValidationMode`
* `mode` => `validationMode`

Reply:
I like that